dashboard/app/jobs/aichat_request_chat_completion_job.rb

```diff
@@ -50,6 +50,14 @@ def perform(request:, locale:)
     request.update!(response: response, execution_status: status)
   end
 
+  private def multimodal?(request)
```

Comment:
Minor: can we move this below around line 110 so it's grouped with the other similar `get_model_id` helper?